windows-installer/generate_sdl_agent_exe.ps1

```diff
@@ -122,13 +107,13 @@ $ProgressPreference = "silentlyContinue"
 # Handle SSL correctly.
 [Net.ServicePointManager]::SecurityProtocol = 'TLS12'
 # Pretend to be curl for Sourceforge redirects to work.
-Invoke-WebRequest "$RUBY_DEV_INSTALLER_LINK" -OutFile "$RUBY_DEV_INSTALLER" -UserAgent "curl/7.60.0"
+Invoke-WebRequest "$RUBY_INSTALLER_LINK" -OutFile "$RUBY_INSTALLER" -UserAgent "curl/7.60.0"
 Invoke-WebRequest "$NSIS_INSTALLER_LINK" -OutFile "$NSIS_INSTALLER" -UserAgent "curl/7.60.0"
 Invoke-WebRequest "$NSIS_UNZU_INSTALLER_LINK" -OutFile "$NSIS_UNZU_ZIP" -UserAgent "curl/7.60.0"
 
 
 ##############################
-#  STEP 3 - INSTALL RUBY.
+#  STEP 3 - INSTALL RUBY AND THE RUBY DEV KIT.
```

Comment:
Looks like we are no longer installing Ruby dev kit? Or is it installed along with Ruby by default now?

Reply:
The new installer contains both Ruby and the devkit.